Review for project-one.

jamestucker1988 opened the pull request:
we made the following changes on TSE's recommendations:
1-fix validation errors.
2-aligned video player with recommended videos.
3-aligned elements in video info section.
4-styled footers to look similar to source page
5-fixed horizontal scroll.
6-add a Readme.md file.
7-have a good and meaningful history of commits.

Dandush03 commented:
# :red_circle: Status Require: changes :muscle: 
### Hey @ricardo123321  :wave: 
## Good attempts on this one :muscle: :+1: 

### This page is a clone page from the [archive youtube](http://archive.vn/Bss88), therefore there are a few changes you have to fix in order to achieve it

## Here are the issues to address to get the project meet the requirement:wink:

## the right panel (Next Videos)
- [ ] your elements are missing some alignments
- [ ] Wrong proportion of width (too much padding maybe?)
##### Required:
![Screenshot from 2020-04-09 09-35-07](https://user-images.githubusercontent.com/51087936/78901654-a69fa080-7a46-11ea-9908-803a2fdf186c.png)
##### Yours: 
![Screenshot from 2020-04-09 09-33-34](https://user-images.githubusercontent.com/51087936/78901662-a8696400-7a46-11ea-9143-8994c2a6d433.png)

## Main video title
- [ ] your elements are missing some alignments
- [ ] 'T' button should be a square and have mo padding
##### Required:
![Screenshot from 2020-04-09 09-34-56](https://user-images.githubusercontent.com/51087936/78901678-b0290880-7a46-11ea-8b02-b6119220b7b1.png)
##### Yours: 
![Screenshot from 2020-04-09 09-34-22](https://user-images.githubusercontent.com/51087936/78901686-b323f900-7a46-11ea-942f-4af16a0db390.png)

## Footer
- [ ] your elements are missing some alignments
- [ ] Wrong proportion of width
- [ ] Wrong Background-color in your elements
- [ ] Wrong color in you text
- [ ] Fix YouTube Logo
##### Required:
![Screenshot from 2020-04-09 09-35-38](https://user-images.githubusercontent.com/51087936/78901707-bb7c3400-7a46-11ea-9b24-61ca18511f87.png)
##### Yours: 
![Screenshot from 2020-04-09 09-35-28](https://user-images.githubusercontent.com/51087936/78901721-bfa85180-7a46-11ea-9af0-945a1aa9e4d8.png)

## Kindly address all these changes and submit a new code review

### Happy coding:computer: 

#### Daniel Laloush

ricardo123321 commented:
Hello, this is not my project

El jue., 9 de abr. de 2020 9:45 a. m., Daniel Laloush <
notifications@github.com> escribió:

> 🔴 Status Require: changes 💪 Hey @ricardo123321
> <https://github.com/ricardo123321> 👋 Good attempts on this one 💪 👍 This
> page is a clone page from the archive youtube <http://archive.vn/Bss88>,
> therefore there are a few changes you have to fix in order to achieve it Here
> are the issues to address to get the project meet the requirement😉 the
> right panel (Next Videos)
>
>    - your elements are missing some alignments
>    - Wrong proportion of width (too much padding maybe?)
>
> Required:
>
> [image: Screenshot from 2020-04-09 09-35-07]
> <https://user-images.githubusercontent.com/51087936/78901654-a69fa080-7a46-11ea-9908-803a2fdf186c.png>
> Yours:
>
> [image: Screenshot from 2020-04-09 09-33-34]
> <https://user-images.githubusercontent.com/51087936/78901662-a8696400-7a46-11ea-9143-8994c2a6d433.png>
> Main video title
>
>    - your elements are missing some alignments
>    - 'T' button should be a square and have mo padding
>
> Required:
>
> [image: Screenshot from 2020-04-09 09-34-56]
> <https://user-images.githubusercontent.com/51087936/78901678-b0290880-7a46-11ea-8b02-b6119220b7b1.png>
> Yours:
>
> [image: Screenshot from 2020-04-09 09-34-22]
> <https://user-images.githubusercontent.com/51087936/78901686-b323f900-7a46-11ea-942f-4af16a0db390.png>
> Footer
>
>    - your elements are missing some alignments
>    - Wrong proportion of width
>    - Wrong Background-color in your elements
>    - Wrong color in you text
>    - Fix YouTube Logo
>
> Required:
>
> [image: Screenshot from 2020-04-09 09-35-38]
> <https://user-images.githubusercontent.com/51087936/78901707-bb7c3400-7a46-11ea-9b24-61ca18511f87.png>
> Yours:
>
> [image: Screenshot from 2020-04-09 09-35-28]
> <https://user-images.githubusercontent.com/51087936/78901721-bfa85180-7a46-11ea-9af0-945a1aa9e4d8.png>
> Kindly address all these changes and submit a new code review Happy coding
> 💻 Daniel Laloush
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jamestucker1988/jamestucker1988microverse_embeded-image-project1-combo/pull/5#issuecomment-611536313>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AONI2U7IW3D2B4CXW643ZD3RLXGP3ANCNFSM4MEXAZYQ>
> .
>

chelmerrox commented:
We have  divided our web page as such:

1. navbar that includes the sign-in button
2. main container that has:
- main container 1 (column 1) with video etc.
- main container 2 (column 2) with recommended videos which is divided into 2 more columns
 a) recommended videos thumbnails
 b) recommended videos titles/texts
3. footer and its elements

We have fixed all of the errors. This is a brief summary:
1. aligned the sidebar and its elements
2. aligned the youtube avatar, subscribe button and number of subscribers
3. styled our footer and its elements to look similar to the source page
4. fixed html validation errors

chelmerrox commented:
We have divided our web page as such:

1. navbar that includes the sign-in button
2. main container that has:
   - main container 1 (column 1) with video etc.
   - main container 2 (column 2) with recommended videos which is divided into 2 more columns
      a) recommended videos thumbnails
      b) recommended videos titles/texts
3. footer and its elements

We have done the following changes to our web page:

1. Added at most 3 icons in the navbar
2. Adjusted the width of the video
3. Adjusted the width and margin (left & right) of the whole body
4. Set a specified width for main container 1 (first column that has video etc)
5. Fixed our footer and its elements

**NOTE: We have tried to replicate exactly this <a href="http://archive.vn/Bss88">web page.</a> We've included this in our README.md description and this is the best that we can achieve. This is why our web page does not look like the updated version of any youtube web page.**

lekegitrepo commented:
### Hi @jamestucker1988 and @chelmerrox 👏 👏
### _You have done a good job 💪 . However, after viewing the link of the page you listed in your comment I discovered that some contents on your page are cut, for example, the "choose your language" section and the text of the listed video on the right side of the page. I guess that's why the previous reviewer asked you to add a margin to the page_

**_Try to fix these and submit for review._**

**_if you have any doubts or questions contact me on slack._**

### _Reviewed by Temitayo Adeleke._

chelmerrox commented:
We have done the following changes:

1. adjusted the size of the Choose language container
2. added margin (left and right) to the whole body of web page
3. re-positioned all other elements affected by the changes above

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

jamestucker1988 commented:
we made the following changes after a recommendation from Tse's-
1)We removed inline styling.
2) Fixed search bar and search magnifying glass to satisfaction.
3) Rearrange buttons in video-info-container.
4)fixed validation errors.
5)Applied Dry rule of coding.
6)gave a common class to the recommended video section.

chelmerrox commented:
We have done the following changes:

1. re-aligned the save and more buttons with the like, dislike and share buttons
2. re-aligned the Upload text in the side container
3. Re-positioned etc any other elements affected by the changes above

chelmerrox commented:
We have done all of the changes recommended by TSEs reviewing our project on the review branch of this repo.

Here is the preview of our webpage which can also be seen in our README.md file:

[screenshot 1](https://github.com/jamestucker1988/jamestucker1988microverse_embeded-image-project1-combo/tree/review/images/webpage_preview_1.PNG)

[screenshot 2](https://github.com/jamestucker1988/jamestucker1988microverse_embeded-image-project1-combo/tree/review/images/webpage_preview_3.PNG)

[screenshot 3](https://github.com/jamestucker1988/jamestucker1988microverse_embeded-image-project1-combo/tree/review/images/webpage_preview_4.PNG)

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

mikenath223 commented on index.html:
```diff
@@ -1,436 +1,214 @@
-                <section class="videotitle-and-info">
-
-                     <!--Loading comments section; code is from the source page: http://archive.vn/Bss88-->
-                     <div style="font-size:100%;background-color: transparent; text-align:center;line-height:20px;white-space:nowrap;border-width: 0px; border-style: none; margin: 0px; padding: 1em; border-color: white; " >
```
Please remove this inline styling and other similar ones too.

jamestucker1988 replied:
THIS is not from our current file. Please check Again. we are on branch review.

mikenath223 commented on styles/style.css:
```diff
@@ -0,0 +1,394 @@
+    justify-content: flex-end;
+}
+
+#search-bar {
+    padding-top: 0.59rem;
+    color: lightgrey;
+    font-size: 1rem;
+    flex: 5 1 60%;
+}
+
+#button {
+    color: grey;
+    background-color: rgb(248, 244, 244);
+    box-shadow: none;
+    height: 2rem;
+    border: 1px solid grey;
+    position: relative;
+    flex: 1 5 5%;
+    padding: 3%;
+}
```
Avoid using ID's for styling. Please change to CSS classes instead.

chelmerrox commented:
We have done all changes recommended by the previous TSEs. We have replicated this exact version of the <a href="http://archive.vn/Bss88">webpage</a> but have been told to follow the <a href="https://www.youtube.com/watch?v=V74l_zS1x8E">updated version</a> of it instead which we have just done. We also realized that in doing this, we are following some Project requirements listed in Pathwright (e.g there should be three icons on the right side of navbar).

**NOTE: Kindly peruse through the previous TSEs comments. We keep getting conflicting feedback on which version of the webpage to replicate. Thanks in advance.**

mikenath223 commented:
Hi Team.
This review request has been marked as invalid

### ✖ INVALID REQUEST

Reason: 

Kindly fix the requested change in the previous review.

I apologize since this is new to you. 

Feel free to contact me on slack(@michgolden) if you need further clarification.

**Welcome to Microverse🚀**
HAPPY CODING👨‍💻

jamestucker1988 commented:
-The feedback response is taken into account and implemented in the current draft of project.
- We modelled a hybrid page with links <a href="http://archive.vn/Bss88">old page</a> and <a href="https://www.youtube.com/watch?v=V74l_zS1x8E">new page</a>. the overall page looks good in shape and all the modelling and cloning have been from these two pages.


**NOTE: PLEASE REFER TO PREVIOUS FEEDBACKS.**

chelmerrox commented:
We have done these changes:

1. Remove all unnecessary branches from local and remote repositories
2. Included the <a href="https://raw.githack.com/jamestucker1988/jamestucker1988microverse_embeded-image-project1-combo/review/index.html">Live demo</a> of our webpage in README.md file
3. Enabled stickler app in the remote repo 
4. Fixed all style errors
5. Adjusted styles etc of any elements affected by the changes above
